ports/libstemmer/portfile.cmake

```
@@ -2,7 +2,7 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://snowballstem.org/dist/libstemmer_c-${VERSION}.tar.gz"
-    FILENAME "libstemmer_c-${VERSION}.tgz"
+    FILENAME "libstemmer_c-${VERSION}.gz"
```

```suggestion
    FILENAME "libstemmer_c-${VERSION}.tar.gz"
```

As in the file in the URL.